V1/Controllers/OrganisationController.cs

```diff
@@ -57,11 +57,11 @@ public async Task<IActionResult> GetOrganisation(string id)
             return StatusCode(resp.Meta.Code, resp);
         }
 
-        [HttpPost("organisations/{id}/check_in")]
-        public async Task<IActionResult> CheckIn(string id, [FromBody] UpdateCountRequest payload)
+        [HttpPost("organisations/{id}/check_in/{isPositive?}")]
```

Comment:
QUESTION: Does `isPositive` best describe the param? Would `{status}` not be a better description?

Reply:
I did see @deltaDirk mention below that you want to test if the user is infected, so this comment might not useful